src/main/java/org/jitsi/videobridge/health/HealthIntervalProperty.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright @ 2018 - present 8x8, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.jitsi.videobridge.health;
+
+import org.jitsi.utils.config.*;
+import org.jitsi.videobridge.util.config.*;
+
+import java.util.concurrent.*;
+
+
+/**
+ * The property which configures the interval between health
+ * checks.
+ */
+public class HealthIntervalProperty extends AbstractConfigProperty<Integer>
+{
+    protected static final String legacyPropName = "org.jitsi.videobridge.health.INTERVAL";
+    protected static final String propName = "videobridge.health.interval";
```

Comment:
In the old code the _PNAME constants were public. Do we want to make the new propName constants public too?

Reply:
i don't think there's any use for that now, as previously we'd sometimes refer to a pname from multiple places and therefore it needed to have potentially public access...now the single source of truth is the property itself, so i think it's best to hide away the detail of the prop name.